corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -108,7 +108,7 @@ def _sql_column(self):
     def get_values_for_query(self, query_context):
         query = self._adapter.session_helper.Session.query(self._sql_column)
         if query_context.query:
-            query = query.filter(self._sql_column.contains(query_context.query))
+            query = query.filter(self._sql_column.ilike(u"%{}%".format(query_context.query)))
```

Comment:
That's what we were hoping for - unfortunately SQLAlchemy doesn't provide that method:
```python
ipdb> self._sql_column.icontains
*** AttributeError: Neither 'Column' object nor 'Comparator' object has an attribute 'icontains'
```
Here's the open request to add that: https://bitbucket.org/zzzeek/sqlalchemy/issues/3482/icontains-istartswith-iendswith